pallets/runtime-interfaces/src/lib.rs

```diff
@@ -40,12 +43,14 @@ pub fn initialize_validator_config(
 
 const ETH_KEY_ID_ENV_VAR: &str = "ETH_KEY_ID";
 const ETH_RPC_URL_ENV_VAR: &str = "ETH_RPC_URL";
+const MATIC_RPC_URL_ENV_VAR: &str = "MATIC_RPC_URL";
 const MINER_ENV_VAR: &str = "MINER";
 const OPF_URL_ENV_VAR: &str = "OPF_URL";
 
 const ETH_KEY_ID_DEFAULT: &str = gateway_crypto::ETH_KEY_ID_ENV_VAR_DEV_DEFAULT;
 const MINER_DEFAULT: &str = "Eth:0x0000000000000000000000000000000000000000";
 const ETH_RPC_URL_DEFAULT: &str = "https://ropsten-eth.compound.finance";
+const MATIC_RPC_URL_DEFAULT: &str = "https://ropsten-eth.compound.finance"; // xxx todo:wn fix this before merge
```

Comment:
Looks like this needs a fix, especially giving the comment

Reply:
Yeah, these defaults aren't really terribly useful anyway. I think I'll just set it to `""` and call it a day.